docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -98,7 +98,7 @@ kubectl delete pod qos-demo --namespace=qos-example
 A Pod is given a QoS class of Burstable if:
 
 * The Pod does not meet the criteria for QoS class Guaranteed.
-* At least one Container in the Pod has a memory or cpu request.
+* At least one Container in the Pod has a memory or cpu request or limit.
```

Comment:
if the user only set limit not request, as a default result, the request value equal the limit value, and that is `guaranteed`.